Sources/Rendering/Core/ImageCPRMapper/example/index.js

```diff
@@ -350,6 +354,54 @@ angleEl.addEventListener('input', () =>
   setAngleFromSlider(radiansFromDegrees(Number.parseFloat(angleEl.value, 10)))
 );
 
+function useStraightenedMode() {
+  mapper.setCenterPoint(null);
+  mapper.setUseUniformOrientation(false);
+  mapper.getOrientedCenterline().setDistanceFunction(vec3.dist);
+  updateDistanceAndDirection();
+}
+
+function useStretchedMode() {
+  mapper.setCenterPoint(centerline.getPoints().getPoint(0));
+  mapper.setUseUniformOrientation(true);
+  mapper.getOrientedCenterline().setDistanceFunction((a, b) => {
```

Comment:
should probably be set automatically by the mapper in setUseUniformOrientation( and not manually by the application

Reply:
No, the distance function depends on which result is expected: a stretched or a projected CPR won't have the same distance function but will both use a uniform orientation
Also, a straightened CPR could use a uniform orientation